pages/best-practices/avo-functions-alongside-existing-tracking.mdx

```diff
@@ -0,0 +1,214 @@
+---
+layout: 'guide'
+title: 'Using Avo Functions Alongside Existing Tracking'
+---
+
+import Link from '../../components/Link';
+
+In this guide we'll cover all the required steps to start using Avo in your app or website, alongside existing tracking code and SDKs.
+
+Integrating Avo with your app or website can begin as soon as you create an Avo account, and requires three steps:
+
+1. [Configure Connections in your Avo workspace](#configure-connections-in-your-avo-workspace)
+2. [Install the Avo CLI](#install-the-avo-cli)
+3. [Initialize Avo and send your first event](#initialize-avo-and-send-your-first-event)
+
+## 1. Configure your Avo workspace
+
+In [your Avo workspace](https://avo.app/welcome) head to the Connections tab in the left sidebar. Here we'll create a Source for our app and a Destination for every SDK we want to pass data into.
+
+> 💡 **What are Connections?** <br />
+> In Avo you define from which apps, websites and servers your data is sent (_Sources_) and to which SDKs and APIs the data is passed into (_Destinations_). In Avo the relation between Sources and Destinations are called Connections.
+
+<br />
+
+### Configure a source
+
+If a source doesn't already exist for your app or website, click "Add Source" to create a new Source. Give it a name and define what development platform and programming language you'll be using. This configuration is used to tailor the Avo Functions to your stack.
+
+<img
+  src={require('../../images/best-practices/avo-functions-alongside-existing-tracking/source-config.png')}
+  alt="Source configuration"
+/>
+<center>
+  <i>
+    A source configuration for MyApp.com, which is a Web app written in
+    TypeScript
+  </i>
+</center>
+
+<br />
+
+### Configure destinations
+
+> 💡 **What is a Custom Destination?** <br />
+> In Avo there are two options when it comes to destinations:
+>
+> 1. Let Avo fully manage the destination for you (everything from initializing to passing data into it), or
+> 2. Connect Avo to a destination (e.g. SDK) you already have initialized and ready to go in your app. This is what we call a Custom Destination.
+
+Now let's create a custom destination for every SDK we want to use in our Source. Head back to the Connections tab and click "Add Destination". To configure a destination:
```

Comment:
I think we should add a quick description why we are going with a custom destinations and not other options.

Reply:
Updated in 4ba6132